gemma-core/src/main/java/ubic/gemma/persistence/persister/CachingPersister.java

```diff
@@ -0,0 +1,10 @@
+package ubic.gemma.persistence.persister;
+
+/**
+ * Special type of {@link Persister} with caching capabilities.
+ * @param <T>
+ */
+public interface CachingPersister<T> extends Persister<T> {
```

Comment:
I noticed that some persisters rely on caching behaviour of other persisters. This interface makes it more consistent.